meveo-admin/ejbs/src/main/java/org/meveo/service/technicalservice/wsendpoint/WebsocketServerEndpoint.java

```diff
@@ -112,6 +115,19 @@ public void onOpen(Session session, EndpointConfig config, @PathParam("endpoint-
 			session.getUserProperties().put("endpointName", endpointName);
 			if (username != null) {
 				session.getUserProperties().put("username", username);
+
+				if(LIQUICHAIN_MODULE_CODE.equals(endpointName)){
```

Comment:
That's not good. You need to find a solution where the module code is not hardcoded

Reply:
Resolved now